konfera/views.py

```diff
@@ -67,8 +67,9 @@ def cfp_form_view(request):
         speaker_instance = speaker_form.save()
         talk_instance = talk_form.save(commit=False)
         talk_instance.primary_speaker = speaker_instance
+        talk_instance.status = 'cfp'
```

Comment:
hardcoding constants is bad practice. Let`s set the constants free in talks.py (please amend also all other lists).

```python
CFP = 'cfp'

TALK_STATUS = (
    (CFP, 'Call For Proposals'),
)
```
and in this view you can import from CFP and set the constant directly. This way we are always sure we use the same constants, and if it changes in future we change it only at one place.